app/controllers/registrations_controller.rb

```diff
@@ -1,4 +1,7 @@
 class RegistrationsController < Devise::RegistrationsController
+
+include RequestsHelper
```

Comment:
@GettyGertie for this [error](https://travis-ci.org/AgileVentures/LocalSupport/builds/347279852?utm_source=github_status&utm_medium=notification) you will have to remove this line, since we are not using the `RequestsHelper` anymore. I think that will solve this error.